cola/themes.py

```diff
@@ -551,8 +570,26 @@ def get_all_themes():
         Theme(
             'flat-dark-green', N_('Flat dark green'), True, EStylesheet.FLAT, '#42a65c'
         ),
+        Theme(
+            'system', N_('System'), None, EStylesheet.SYSTEM, None
+        ),
     ]
 
+    # check if themes path exists in user folder
+    path = resources.config_home('themes')
+    if not os.path.exists(path):
+        return themes
+
+    # get only files with extension .qss
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            name, ext = os.path.splitext(file)
+            if ext == '.qss':
+                themes.append(
+                    Theme(name, N_(name.capitalize()), None, EStylesheet.CUSTOM, None)
```

Comment:
Is it okay if we didn't do the `name.capitalize()` thing?

Reply:
The 3rd parameter should be `False` not `None`. `is_dark` is a bool.

Reply:
removed capitalize, changed to False